sdk/core/core-lro/README.md

```diff
@@ -1,18 +1,24 @@
 # Azure Core LRO client library for JavaScript
 
-Azure's Core LRO is a JavaScript library that provides an API that aims to
-allow the azure-sdk-for-js public libraries to implement fully featured pollers
-to manage Long Running Operations (LROs) with the Azure services.
+`@azure/core-lro` is a JavaScript library that allows developers to build pollers that keep track of Long Running Operations (LROs) against Azure services. Until completion, such operations require consecutive calls to Azure services to update a local representation of the remote operation status.
 
-@azure/core-lro is made following our guidelines here: https://azure.github.io/azure-sdk/typescript_design.html#ts-lro
+**Please note:** This library is intended to be used as a dependency of official Azure SDK packages and not intended for inclusion in your projects directly.
```

Comment:
```suggestion
**Please note:** This library is included with other Azure SDK libraries that need it. It should not be used as a direct dependency in your projects.
```

Want to be more clear that users shouldn't be manually npm-installing this thing.